src/HDC1080.c

```diff
@@ -63,7 +72,13 @@ float GetHumidity(void)
 	uint8_t regaddr = HDC1080_HUM_REG;
 	uint8_t regval[2];
 
-	I2C_IF_ReadFrom(HDC1080_DEV_ADDR, &regaddr, 1, regval, 2);
+	I2C_IF_Write(HDC1080_DEV_ADDR, &regaddr, 1, 0);
+
+	// TODO: Change to OS based delay, and add clarity to time param
+	MAP_UtilsDelay((60*30*1000)); // 7.5msec
+
+	I2C_IF_Read(HDC1080_DEV_ADDR, regval, 2);
+
 	// TODO: Handle error case here.
 
 	uint16_t res = (regval[0] << 8) | regval[1];
```

Comment:
Simplest design first. Until we see a need for it, the primary functionality of sampling humidity and temperature is the most important for now.